kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java

```diff
@@ -0,0 +1,196 @@
+package com.applozic.mobicomkit.uiwidgets.conversation.fragment;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.applozic.mobicomkit.uiwidgets.R;
+import com.applozic.mobicommons.commons.core.utils.Utils;
+
+/**
+ * fragment for the feedback input form
+ * @author shubham
+ * @date july 19
+ */
+public class FeedbackInputFragment extends Fragment {
+
+    private static final String TAG = "FeedbackInputFragment";
+
+    EditText editTextFeedback;
+    Button button1;
+    Button button2;
+    Button button3;
+    TextView textView1;
+    TextView textView2;
+    TextView textView3;
+    Button buttonSubmit;
+    Button closeButton;
+
+    FeedbackFragmentListener feedbackFragmentListener;
+    static final int RATINGBAD = 1;
+    static final int RATINGGOOD = 2;
+    static final int RATINGGREAT = 3;
+    private int ratingValue;
+
+    public void setFeedbackFragmentListener(FeedbackFragmentListener feedbackFragmentListener) {
+        this.feedbackFragmentListener = feedbackFragmentListener;
+    }
+
+    public FeedbackInputFragment() {
+        super();
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        if (!FeedbackFragmentListener.class.isAssignableFrom(MobiComConversationFragment.class)) {
+            Utils.printLog(context, TAG, "Implement FeedbackFragmentListener in your parent fragment.");
+            throw new ClassCastException("Implement FeedbackFragmentListener in your parent fragment.");
+        }
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.feedback_rating_layout, container, false);
+
+        editTextFeedback = view.findViewById(R.id.idEditTextFeedback);
+        buttonSubmit = view.findViewById(R.id.idButtonSubmit);
+        button1 = view.findViewById(R.id.idButton1);
+        button2 = view.findViewById(R.id.idButton2);
+        button3 = view.findViewById(R.id.idButton3);
+        textView1 = view.findViewById(R.id.idText1);
+        textView2 = view.findViewById(R.id.idText2);
+        textView3 = view.findViewById(R.id.idText3);
+        closeButton = view.findViewById(R.id.idCloseFeedbackFragment);
+
+        button1.setScaleX(0.8f);
+        button2.setScaleX(0.8f);
```

Comment:
What's 0.8

Reply:
The scale of the image is reduced initally so that it can be bought back to normal when selected.

Reply:
replace it with constant